### DISCUSSION TITLE
Importing boolean values and floats in the same table

### DISCUSSION THREAD

**Discussion by Mastomaki** [Spine-project/Spine-Toolbox#2576] [Support]
Hello!

Using the importer tool. Is it possible to import boolean values and floats into parameter values from the same column? As one should choose either float or boolean value.

![image](https://github.com/spine-tools/Spine-Toolbox/assets/42839563/9f46954b-8410-413a-9ca7-7bf6155d5411)

**Comment by Mastomaki** [comment]
In this case one can select the **string** type.

**Reply by soininen** [reply]
Do you mean that Importer can cast the values to string and floats properly if you select string as the column type?

**Reply by Mastomaki** [reply]
Yes.

**Reply by soininen** [reply]
Nice! I didn't remember it worked that way even though I've dived deep into the import code itself.
